src/test/java/plugins/LdapPluginTest.java

```diff
@@ -246,6 +247,7 @@ public void enable_cache() throws IOException {
     }
     
     @Test
+    @Ignore("Regression - Environment variables are not working. Test is failing.")
```

Comment:
Why is this ignored? If that is a regression in the plugin it should fail...

Reply:
@olivergondza my intention when ignoring the test is to avoid confusion when it runs and fails. Once the new release of the plugin is published and the regression is fixed, I will give up ignoring it.

Reply:
Please, un-ignore it. If there is a regressions and you spend the time to implement a test for that, we want it to know the version is affected.